lib/datadog/appsec/configuration/settings.rb

```diff
@@ -33,10 +33,9 @@ def self.add_settings!(base)
           base.class_eval do
             settings :appsec do
               option :enabled do |o|
-                o.default { env_to_bool('DD_APPSEC_ENABLED', DEFAULT_APPSEC_ENABLED) }
-                o.setter do |v|
-                  v ? true : false
-                end
+                o.type :bool
+                o.env 'DD_APPSEC_ENABLED'
+                o.default DEFAULT_APPSEC_ENABLED
```

Comment:
This feels weird, seems like it should read `o.default true`.

Indeed whatever else depends on `DEFAULT_APPSEC_ENABLED` should instead read the configuration setting.

Reply:
This is legacy from when the appsec setting was not using the core configuration code. 

We can remove all those constants on a separate PR 😄